docs/docs/recipes/create-your-connector/connector-file-structure.mdx

```diff
@@ -11,9 +11,7 @@ Files whose name ends up with `^` refer to an optional file, otherwise are requi
 
```

Comment:
for PR title and commit: will it be better to use "docs" / "documentation" instead of  "documents"?